common/containers/Tabs/TransactionStatus/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import TabSection from 'containers/TabSection';
+import {
+  signLocalTransactionSucceeded,
+  TSignLocalTransactionSucceeded,
+  signTransactionFailed,
+  TSignTransactionFailed
+} from 'actions/transaction';
+import { computeIndexingHash } from 'libs/transaction';
+import translate from 'translations';
+import SimpleButton from 'components/ui/SimpleButton';
+import EthTx from 'ethereumjs-tx';
+import classnames from 'classnames';
+import { toBuffer } from 'ethereumjs-util';
+import { getSerializedTransaction } from 'selectors/transaction';
+import { AppState } from 'reducers';
+import './index.scss';
+
+interface StateProps {
+  stateTransaction: AppState['transaction']['sign']['local']['signedTransaction'];
```

Comment:
We actually would not use a signed transaction to check its status. We instead want to use the transaction hash that comes back from a broadcasted transaction. Transaction hashes look kind of like addresses, `0x414ab421...` and are just pointers to a transaction. Signed transactions are what we send to broadcast the transaction, and has all of the transaction data encoded into it.